src/schema/schema_create.c

```diff
@@ -961,6 +977,103 @@ __create_data_source(
     return (dsrc->create(dsrc, &session->iface, uri, (WT_CONFIG_ARG *)cfg));
 }
 
+/*
+ * __import_entry_cmp --
+ *     Qsort function: sort the import entries array by name.
+ */
+static int WT_CDECL
+__import_entry_cmp(const void *a, const void *b)
+{
+    WT_IMPORT_ENTRY *ae, *be;
+
+    ae = (WT_IMPORT_ENTRY *)a;
+    be = (WT_IMPORT_ENTRY *)b;
+
+    return strcmp(ae->name, be->name);
+}
+
+/*
+ * __get_uri_suffix --
+ *     Get suffix of the URI.
+ */
+static int
+__get_uri_suffix(WT_SESSION_IMPL *session, const char *uri, const char **suffix)
+{
+    *suffix = strchr(uri, ':');
+    if (*suffix == NULL || (++(*suffix))[0] == '\0')
+        WT_RET_MSG(session, WT_ERROR, "%s invalid URI", uri);
+
+    return (0);
+}
+
+/*
+ * __metadata_entry_worker --
+ *     Worker function for metadata file reader procedure. The function populates import list with
+ *     entries related to the import URI.
+ */
+static int
+__metadata_entry_worker(WT_SESSION_IMPL *session, WT_ITEM *key, WT_ITEM *value, void *state)
```

Comment:
Again, prefix this function with `__create_`

Reply:
Renamed to `__create_meta_entry_worker`.